gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/PubSubOptions.java

```diff
@@ -31,7 +31,7 @@
   private static final long serialVersionUID = 5640180400046623305L;
   private static final String PUBSUB_SCOPE = "https://www.googleapis.com/auth/pubsub";
   private static final Set<String> SCOPES = ImmutableSet.of(PUBSUB_SCOPE);
-  private static final String DEFAULT_HOST = "https://pubsub.googleapis.com";
+  private static final String DEFAULT_HOST = "pubsub.googleapis.com";
```

Comment:
I dropped this as part of #1149. Pub/Sub is over gRPC so no need for `https://`.